shared/locales/activities/activities.en-US.properties

```diff
@@ -1,31 +1,29 @@
 # Default activities label
 
-# PICK
-default-activity-pickimage    = Take picture
-
 # DIAL
-default-activity-makecall     = Make call
+default-activity-dialnumber     = Make call
 
 # NEW
-default-activity-opencontact  = Open contact
-default-activity-sendmail     = Send email
-default-activity-sendmessage  = Send message
+default-activity-newcontact     = Open contact
```

Comment:
Is this correct? From a key called newcontact I'd expect something like "Create contact"